rules/aep0136/aep0136.go

```diff
@@ -51,7 +51,7 @@ func isCustomMethod(m *desc.MethodDescriptor) bool {
 
 	// Methods with no `:` in the URI are standard methods if they begin with
 	// one of the standard method names.
-	for _, prefix := range []string{"Get", "List", "Create", "Update", "Delete", "Replace"} {
+	for _, prefix := range []string{"Get", "List", "Create", "Update", "Delete", "Replace", "Apply"} {
```

Comment:
Apply technically isn't in the AEPs yet - but I'm fine with adding it! or just commenting out that line from aepc (feel free to resolve this comment either way)